presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -317,6 +326,11 @@ public StreamReader getStreamReader(int index)
         return streamReaders[index];
     }
 
+    public Map<String, Slice> getUserMetadata()
+    {
+        return userMetadata;
```

Comment:
Yeah. 
This kinda sucks. 
Maybe we should provide some `UserMetadataDecoder` when we construct `OrcRecordReader` and never expose the original `Map<String, Slice>` its users?

Copying for now.